website/docs/guides/core-to-cloud-1.md

```diff
@@ -0,0 +1,244 @@
+
+3. Configure [Single Sign-On (SSO)](/docs/cloud/manage-access/sso-overview) or [Role-based access control (RBAC)](/docs/cloud/manage-access/about-user-access#role-based-access-control) for easy and secure access. <Lifecycle status='enterprise' />
+   - This removes the need to save passwords and secret environment variables locally.
+
+4. In **Account settings**, switch on [partial parsing](/docs/deploy/deploy-environments#partial-parsing) to only reparse changed files, saving time.
+
+5. In **Account settings**, enable [Git repo caching](/docs/deploy/deploy-environments#git-repository-caching) to ensure job reliability, protection from third-party outages, and performance. <Lifecycle status='enterprise' />
+
+## Data platform setup
+
+This section explains the considerations and methods to connect your data platform to dbt Cloud.
+
+1. Set up your [data platform connections](/docs/cloud/connect-data-platform/about-connections) and [environment variables](/guides/core-to-cloud-1?step=4) in dbt Cloud. dbt Cloud can connect with a variety of data platform providers including:
+   - [AlloyDB](/docs/cloud/connect-data-platform/connect-redshift-postgresql-alloydb) 
+   - [Amazon Redshift](/docs/cloud/connect-data-platform/connect-redshift-postgresql-alloydb) 
+   - [Apache Spark](/docs/cloud/connect-data-platform/connect-apache-spark) 
+   - [Databricks](/docs/cloud/connect-data-platform/connect-databricks) 
+   - [Google BigQuery](/docs/cloud/connect-data-platform/connect-bigquery)
+   - [Microsoft Fabric](/docs/cloud/connect-data-platform/connect-microsoft-fabric)
+   - [PostgreSQL](/docs/cloud/connect-data-platform/connect-redshift-postgresql-alloydb)
+   - [Snowflake](/docs/cloud/connect-data-platform/connect-snowflake)
+   - [Starburst or Trino](/docs/cloud/connect-data-platform/connect-starburst-trino)
+
+2. You can verify your data platform connections by clicking the **Test connection** button in your deployment and development credentials settings.
+
+### Advanced configuration
+3. Use [OAuth connections](/docs/cloud/manage-access/set-up-snowflake-oauth), which enables secure authentication using your data platform’s SSO.  <Lifecycle status='enterprise' />
+
+## Git setup
+
+Your existing dbt project source code should live in a Git repository. In this step, you should connect your existing dbt project source code from Git to dbt Cloud.
+
+1. Ensure your dbt project is in a Git repository
+
+2. Once you’ve set up a dbt Cloud account, you can [connect and configure Git](/docs/cloud/git/git-configuration-in-dbt-cloud) in dbt Cloud to connect your Git repository:
+   - Connect with one of the [native integrations](/docs/cloud/git/git-configuration-in-dbt-cloud) in dbt Cloud (such as GitHub, GitLab, and Azure DevOps).
+   - [Import a git repository](/docs/cloud/git/import-a-project-by-git-url) from any valid git URL that points to a dbt project.
+
+### Advanced configuration
```

Comment:
Same feedback as above - putting it under a different header makes it look out of place
```suggestion
3. (Advanced configuration) Log into dbt Cloud using [OAuth connections](/docs/cloud/git/connect-github) to integrate with your source code platform. It automatically links to the repository using one of the native integrations set at the account level. <Lifecycle status='enterprise' />
```